onadata/apps/logger/models/xform.py

```diff
@@ -823,10 +823,18 @@ def _set_title(self):
                 _("Title shouldn't have any invalid xml "
                   "characters ('>' '&' '<')"))
 
-        if re.search(r"([://.]+)", self.title):
+        # Capture urls within form title
+        if re.search(r"^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$", self.title):  # noqa
             raise XLSFormError(
                 _("Invalid title value; value shouldn't match a URL"))
 
+        # Capture invalid characters within form title
+        if re.search(r"([://.]+)", self.title):
```

Comment:
This is issue just requires an update on the url regex. I think the title can have those characters